paperclip-storage-ftp.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("rake")
   gem.add_development_dependency("daemon_controller", ">= 1.1.0")
   gem.add_development_dependency("activerecord")
+  gem.add_development_dependency("byebug") unless RUBY_PLATFORM =~ /java/
```

Comment:
I would prefer to remove it. Especially since using `RUBY_PLATFORM` in a gemspec will only have an effect at the time the gem is being built, not at installation time:
http://stackoverflow.com/questions/8940271/build-a-ruby-gem-and-conditionally-specify-dependencies